crates/worker/src/partition/mod.rs

```diff
@@ -123,6 +127,7 @@ where
         networking: Networking<T>,
         bifrost: Bifrost,
         mut partition_store: PartitionStore,
+        configuration: impl LiveLoad<Configuration> + Send + 'static,
```

Comment:
You might have much better time if you use `Live<Configuration>` instead. It's a concrete type so you won't need to box it below.